tests/main/microk8s-smoke/task.yaml

```diff
@@ -34,7 +42,13 @@ execute: |
     #microk8s enable dashboard dns registry istio
     microk8s kubectl get nodes | MATCH Ready
     microk8s kubectl get services | MATCH kubernetes
-    
-    microk8s kubectl create deployment nginx --image=nginx
+
+    echo "Deploy and expose an nginx service"
+    microk8s kubectl run nginx --image=nginx
     retry -n 40 sh -c "microk8s kubectl get pods | MATCH Running"
-    # XXX: test if ngix can be connected
+
+    microk8s kubectl expose pod/nginx --name nginx-np --type NodePort --target-port 80 --port 80
+    microk8s kubectl patch svc nginx-np --patch '{"spec":{"ports":[{"port":80,"nodePort":31313}]}}'
```

Comment:
Curious why this is needed, maybe a short comment?

Reply:
Added.